src/lib/components/Accordion/Accordion.tsx

```diff
@@ -44,10 +44,19 @@ const AccordionComponent: FC<AccordionProps> = ({
   ...props
 }) => {
   const [isOpen, setOpen] = useState(collapseAll ? -1 : 0);
+
+  const toggleOpen = (i: number) => {
+    if (isOpen === i) {
```

Comment:
```ts
const toggleOpen = (i: number) => setOpen(isOpen === i ? -1 : i);
```

Reply:
thanks @rluders  re-ran prettier and changed toggleOpen to a ternary. Let me know anything else needs updating!

Reply:
Yup. Tested locally and reran the tests using the "test:open" script.

Reply:
Let me try it out. I was getting that error for a second and then I stopped getting it, but I'll check out this different syntax and see if that resolves the issue.

Appreciate the feedback!

Reply:
Fix has been pushed. LMK if it looks better. Thanks for your patience!